C/services/south/ingest.cpp

```diff
@@ -28,6 +28,76 @@ static void ingestThread(Ingest *ingest)
 	}
 }
 
+/**
+ * Fetch all asset tracking tuples from DB and populate local cache
+ *
+ * @param m_mgtClient	Management client handle
+ */
+void Ingest::populateAssetTrackingCache(ManagementClient *m_mgtClient)
+{
+	try {
+		std::vector<AssetTrackingTuple*>& vec = m_mgtClient->getAssetTrackingTuples();
+		for (AssetTrackingTuple* & rec : vec)
+			{
+			if (rec->m_serviceName != m_serviceName || rec->m_pluginName != m_pluginName || rec->m_eventName != "Ingest")
```

Comment:
The REST API allows filtering, so would be more efficient to filter on the API call rather than retrieve all the asserts and then filter here.

Reply:
ok sure.